example/src/app/ng-reflect-as-text/ng-reflect-as-text.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-ng-reflect-as-text',
+  templateUrl: './ng-reflect-as-text.component.html'
+})
+export class NgReflectAsTextComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```

Comment:
I know, it comes preconfigured like this, but I like it minimal. Let's strip off the constructor and remove `OnInit` here, so later the code becomes more clear what it is about.

Reply:
Done